src/lambda/commands/copyLambdaUrl.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DefaultLambdaClient, LambdaClient } from '../../shared/clients/lambdaClient'
+import { LambdaFunctionNode } from '../explorer/lambdaFunctionNode'
+import globals from '../../shared/extensionGlobals'
+import { copyToClipboard } from '../../shared/utilities/messages'
+import { addCodiconToString } from '../../shared/utilities/textUtilities'
+
+export const NO_URL_INFO_MSG = 'No URL for Lambda function.'
```

Comment:
We've been avoiding all caps for identifiers:
```suggestion
export const noUrlMessage = 'No URL for Lambda function.'
```

Also, this doesn't need an identifier if we don't export it. It's ok to inline strings directly if they're only used once. This makes things easier to read.

Reply:
Removed this const in favor of putting it inline